.github/workflows/chart_update_on_merge.yml

```diff
@@ -0,0 +1,62 @@
+name: chart-update-on-merge
+
+on:
+  push: # There is no specific "PR merged" event but pushes to protected branch `develop` would only happen during PR merges (and releases) (and Jeff)
+    branches:
+      - develop
+jobs:
+  chart-update:
+    name: Cromwhelm Chart Auto Updater
+    runs-on: self-hosted # Faster machines; see https://github.com/broadinstitute/cromwell/settings/actions/runners
+    steps:
+    - name: Clone Cromwell
+      uses: actions/checkout@v2
+      with:
+        repository: broadinstitute/cromwell
+        token: ${{ secrets.BROADBOT_GITHUB_TOKEN }} # Has to be set at checkout AND later when pushing to work
+        path: cromwell
+    - uses: olafurpg/setup-scala@v10
+      with:
+        java-version: adopt@1.11
+    - name: Clone Cromwhelm
+      uses: actions/checkout@v2
+      with:
+        repository: broadinstitute/cromwhelm
+        token: ${{ secrets.BROADBOT_GITHUB_TOKEN }} # Has to be set at checkout AND later when pushing to work
+        path: cromwhelm
+    - name: Find Cromwell short SHA
+      run: |
+        cd cromwell
+        echo "CROMWELL_SHORT_SHA=`git rev-parse --short $GITHUB_SHA`" >> $GITHUB_ENV
+    - name: Find Cromwell release number
+      run: |
+        previous_version=$(curl -X GET https://api.github.com/repos/broadinstitute/cromwell/releases/latest | jq .tag_name | xargs)
```

Comment:
Feels potentially a bit brittle... not a huge problem now, but will we get alerts if this pattern breaks and the pushes start failing? Or will it be silent failures until we notice cromwhelm has fallen super out of date?

Reply:
Robustified with a regex check that exits on failure